components/clp-package-utils/clp_package_utils/controller.py

```diff
@@ -201,16 +214,15 @@ def _set_up_env_for_queue_bundling(self) -> EnvVarsDict:
         """
         component_name = QUEUE_COMPONENT_NAME
 
-        if BundledService.QUEUE not in self._clp_config.bundled:
+        if self._clp_config.queue is None or BundledService.QUEUE not in self._clp_config.bundled:
             logger.info(
-                "%s is not included in the 'bundled' configuration, skipping service bundling...",
+                "%s is not configured or part of the 'bundled' configuration, skipping "
+                "service bundling...",
                 component_name,
             )
             # Bundling
             return EnvVarsDict({"CLP_QUEUE_ENABLED": "0"})
 
```

Comment:
was the removal unintentional?

```suggestion

        logger.info("Setting up environment for bundling %s...", component_name)

```